src/features/unmarkedWsNoticeList/utils.ts

```diff
@@ -0,0 +1,78 @@
+import { ApplicationStatus } from '../../@types/__generated__/globalTypes';
+import {
+  UNMARKED_WINTER_STORAGE_NOTICES,
+  UNMARKED_WINTER_STORAGE_NOTICES_winterStorageNotices_edges as UNMARKED_WINTER_STORAGE_NOTICES_EDGE,
+  UNMARKED_WINTER_STORAGE_NOTICES_winterStorageNotices_edges_node as UNMARKED_WINTER_STORAGE_NOTICES_NODE,
+} from './__generated__/UNMARKED_WINTER_STORAGE_NOTICES';
+
+interface UnmarkedWinterStorageChoice {
+  winterStorageAreaName: string;
+  winterStorageArea: string;
+}
+
+export type UnmarkedWinterStorageNotice = {
+  boatLength: number;
+  boatModel: string;
+  boatName: string;
+  boatRegistrationNumber: string;
+  boatType?: string | null;
+  boatWidth: number;
+  choice: UnmarkedWinterStorageChoice;
+  createdAt: string;
+  customerId?: string;
+  firstName: string;
+  id: string;
+  lastName: string;
+  status: ApplicationStatus;
+};
+
+export const getUnmarkedWinterStorageNotices = (
+  data: UNMARKED_WINTER_STORAGE_NOTICES | undefined
+): UnmarkedWinterStorageNotice[] => {
+  return (
+    data?.winterStorageNotices?.edges.reduce<UnmarkedWinterStorageNotice[]>((acc, edge) => {
+      const {
+        boatLength,
+        boatModel,
+        boatName,
+        boatRegistrationNumber,
+        boatType,
+        boatWidth,
+        createdAt,
+        firstName,
+        id,
+        lastName,
+        status,
+        winterStorageAreaChoices: choices,
+      } = (edge as UNMARKED_WINTER_STORAGE_NOTICES_EDGE).node as UNMARKED_WINTER_STORAGE_NOTICES_NODE;
```

Comment:
Why do we need to assert the types here?
Aren't these types generated automatically based on the schema?

Reply:
`test:coverage` will whine if we don't either
1. assert these types
2. test the cases where the data for these is invalid and impossible in practice, but still possible based on the schema

Reply:
I will still lean towards 2 (I had a bad experience with hidden bugs due to type assertions) but if that's too much of a hassle, this looks fine.